style.css

```diff
@@ -1,554 +1,1111 @@
+  font-weight: bolder;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 1rem;
 }
 
-.contact form {
-  font-family: 'Inter', sans-serif;
+form {
+  width: 100%;
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  margin-bottom: 0;
-  width: 21.6rem;
 }
 
-input[type="text"],
-input[type="email"] {
+.formel {
+  width: 100%;
+  margin-bottom: 24px;
+  height: 80px;
+  font-size: 20px;
   font-family: 'Inter', sans-serif;
-  font-size: 15px;
-  font-weight: 400;
-  line-height: 24px;
-  letter-spacing: 0;
-  text-align: left;
-  color: #ff6b00;
+  padding-left: 10px;
   border: none;
   border-bottom: 1px solid #dbd8d7;
-  padding: 1rem;
-  width: 17.2rem;
-  margin: 1rem 0;
 }
 
-.footericon {
-  display: flex;
-  position: relative;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  padding: 0;
-  margin-top: 4rem;
-  margin-bottom: 0;
-  border: 2px solid var(--navcolor);
-  border: none;
-  border-top: 1px solid #dfe1e6;
+.textarea {
+  width: 100%;
+  height: 200px;
+  margin-bottom: 36px;
+  font-size: 20px;
+  padding-left: 10px;
 }
 
-textarea {
+footer div.icons form > button {
+  padding: 20px 5px;
+  font-size: 20px;
+  background-color: #ff6b00;
+  color: #fff;
   border: none;
-  width: 19.2rem;
-  height: 7rem;
-  background: var(--txtcolor);
-  font-family: 'Inter', sans-serif;
-  font-size: 15px;
-  line-height: 24px;
-  letter-spacing: 0;
-  text-align: left;
-  font-weight: 400;
-  color: var(--bkcolor);
-  padding-top: 1.5rem;
-  margin-top: 0.4rem;
-  background-color: #fbf8f7;
-  padding-left: 12px;
+  text-align: center;
+  width: 50%;
+  margin-left: 27%;
+  margin-bottom: 60px;
 }
 
-form > input[type="submit"] {
-  margin-top: 2rem;
-  border: none;
-  background: var(--maincolor);
-  color: var(--txtcolor);
-  font-family: 'Inter', sans-serif;
-  font-size: 17px;
-  font-weight: 700;
-  line-height: 24px;
-  letter-spacing: 0.03em;
-  text-align: center;
-  height: 3rem;
-  width: 10rem;
+.line {
+  display: none;
 }
 
-.contact > form > input[type="submit"]:hover {
-  background: var(--btncol2);
-  border: 1px solid var(--maincolor);
-  color: var(--maincolor);
+.recent-work button:hover,
+.work-container > div button:hover,
+.about-container div > button:hover,
+div.icons form > button:hover {
+  color: #ff6b00;
+  border: 1px solid #f55800;
+  background: #fff5e1;
+  transition: background-color ease-in;
+  transition-duration: 1s;
 }
 
-.line1 {
-  width: 6rem;
-  height: 3px;
-  border: none;
-  border-radius: 10px;
-  padding-bottom: 0;
-  background: var(--line);
+body section.recent-work button:focus,
+body section.work-container > div button:focus,
+body section.about-container div > button:focus,
+footer div.icons form > button:focus {
+  background-color: #e05304;
+  color: #f4f1f0;
 }
 
-/* all the display none for mobile */
-nav > :nth-child(3) {
-  display: none;
+@media (min-width: 320px) {
```

Comment:
- [ ] **[OPTIONAL]** Good job doing various @media queries in your styles, but we could separate them into two groups of devices 768px or wider. More information about it, [here](https://www.w3schools.com/cssref/css3_pr_mediaquery.asp).